pkg/cmd/hgctl/dashboard.go

```diff
@@ -179,6 +196,32 @@ func consoleDashCmd() *cobra.Command {
 	return cmd
 }
 
+// accessDocker access docker container
+func accessDocker(cmd *cobra.Command) error {
+	dockerCli, err := command.NewDockerCli(command.WithCombinedStreams(os.Stdout))
+	if err != nil {
+		return fmt.Errorf("build docker CLI client fail: %w", err)
+	}
+	err = dockerCli.Initialize(flags.NewClientOptions())
+	if err != nil {
+		return fmt.Errorf("docker client initialize fail: %w", err)
+	}
+	apiClient := dockerCli.Client()
+	list, err := apiClient.ContainerList(context.Background(), types2.ContainerListOptions{})
+	for _, container := range list {
+		for i, name := range container.Names {
+			if strings.Contains(name, "higress-console") {
+				port := container.Ports[i].PublicPort
+				// not support define ip address
+				url := fmt.Sprintf("http://localhost:%d\n", port)
```

Comment:
now can't open the browser because there is an enter character at the end of url

Reply:
I tested that it is possible to open the browser on my end, but removing it is also possible. The printing information is in the openBrowser method.

Reply:
it can't  open the browser on my MAC

Reply:
Can you open the browser now?

Reply:
it's ok now